src/mock/trip.js

```diff
@@ -0,0 +1,45 @@
+import {getRandomInteger} from 'utils';
+import {TRIP_TYPES, CITIES, DESCRIPTION, PICTURE_URL} from './const.js';
+import {getRandomElement} from 'utils';
+import dayjs from 'dayjs';
+
+const createTrip = () => {
+  return {
```

Comment:
можно написать вот так 
```suggestion
const createTrip = () =>  ({
  твой объект
  )}

```